apps/docs/content/guides/auth/migrations/auth0.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: 'Migrate from Auth0 to Supabase Auth'
+subtitle: 'Learn how to migrate your users from Auth0'
+breadcrumb: 'Auth0 migration'
+---
+
+## Migration Strategies
+
+When it comes to migrating your users to a different authentication service, these are some of the considerations that we've seen across other projects.
```

Comment:
> these are some of the considerations that we've seen across other projects.

these are some of the considerations common across supabase projects: 

(primarily to remove "we")